letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -500,6 +500,7 @@ Bootstrap() {
     echo "You will need to bootstrap, configure virtualenv, and run pip install manually."
     echo "Please see https://letsencrypt.readthedocs.org/en/latest/contributing.html#prerequisites"
```

Comment:
I guess the reason I think they're related is that (in my original thinking when writing this code without an `exit`) if the user is going to have to hack things together manually, it's most helpful if the script keeps running and hits a real error that tells them what dependency they're missing, rather than a generic "bootstrap failed" one.  But that could well be wrong.